user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -174,6 +177,8 @@ Once you have established a trust relationship between DataMiner (i.e. the servi
 
       > [!NOTE]
       > From DataMiner 10.1.11/10.2.0 onwards, DataMiner supports Azure AD application querying. If this is used instead of delegated querying, an authentication secret will suffice and no username and password will need to be specified here.
+      >
+      > After start-up from DataMiner, since clientSecret and password attributes are sensitive data, the necessary precautions are done to the *DataMiner.xml* file.
```

Comment:
This is rather vague. Could you rephrase to make more clear what happens? (E.g. "... the client secret and password are encrypted in DataMiner.xml", if that is what is done.). I'd also move this note to the end of this subprocedure, i.e. after "Save the file and restart DataMiner."